.gitignore

```diff
@@ -1,3 +1 @@
-/subprojects/*
-!/subprojects/packagefiles
-!/subprojects/*.wrap
+/subprojects
```

Comment:
please don't revert that. Just keep it. Or remove also `/subprojects`

Reply:
We probably wouldn't use it but yeah I guess it's better to just keep it the way it was before.